Dockerfile

```diff
@@ -311,6 +313,8 @@ ENV GOMODCACHE=/.cache/mod
 ENV PROTOTOOL_CACHE_PATH=/.cache/prototool
 ARG SOURCE_DATE_EPOCH
 ENV SOURCE_DATE_EPOCH=${SOURCE_DATE_EPOCH}
+# Go standard library is shipped with Talos, thus it must be tracked in SBOM
+RUN mkdir -p /rootfs/usr/share/spdx/ && cp /usr/share/spdx/golang.spdx.json /rootfs/usr/share/spdx/golang.spdx.json
```

Comment:
can't we just do a `COPY --from=tools` as a step in initramfs step, also avoids `RUN` commands which speeds up builds

Reply:
Perhaps a `COPY --from=tools` here, to make this a shared layer and avoid doing this thrice for two initramfs and container rootfs. Will fix, thx